bench/wtperf/wtperf.c

```diff
@@ -753,14 +785,18 @@ run_mix_schedule(CONFIG *cfg, WORKLOAD *workp)
 	memset(workp->ops, WORKER_READ, sizeof(workp->ops));
 
 	pct = (workp->insert * 100) /
-	    (workp->insert + workp->read + workp->update);
+	    (workp->insert + workp->read + workp->truncate + workp->update);
```

Comment:
I follow. I've added logic to make the truncate thread a single exclusive thread within its own workload.